gprofiler/main.py

```diff
@@ -824,29 +871,31 @@ def main() -> None:
         )
 
         set_enrichment_options(enrichment_options)
-
-        gprofiler = GProfiler(
```

Comment:
If we change it to be a subcommand, this diff will be eliminated because we won't need to change anything here